### PR TITLE
Bugfix limited team members

### PR DESCRIPTION
**Pull request by akriti12negi**
#1808 

the team member column should display only limited team members in a single row 

 I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
left some comments, please check.

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -76,7 +76,7 @@
                                     <td class="w-33p"><div class="pl-2 pl-xl-3">{{ $project->name }}</div></td>
                                 @endcan
                                 {{-- <td class="w-33p"></td> --}}
-                                <td>
+                                <td style="width:21%;">
```
[the remark this answers is not in the document]
Comment (P4NK4J) [reply Modules/Project/Resources/views/index.blade.php:79]: @akriti12negi UI looks good. can we have a small space between the member icons in the next row? Also, let's put the style into app.scss rather

[the remark this answers is not in the document]
Comment (akriti12negi) [reply Modules/Project/Resources/views/index.blade.php:79]: okay I'll do the recommended changes @P4NK4J

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**